### PR TITLE
Added basic detection of Netonix

### LINKED ISSUE #1466: Add support for Netonix WISP switches
Hi
Please can we add support for Netonix wisp switches?

http://forum.netonix.com/firmware/wisp-switch-mib
root@zabbix:/home/tim# snmpwalk -v 2c -c BHNetworks -M NETONIX-SWITCH-MIB  172.16.201.1
iso.3.6.1.2.1.1.1.0 = ""
iso.3.6.1.2.1.1.2.0 = OID: iso.3.6.1.4.1
iso.3.6.1.2.1.1.3.0 = Timeticks: (298143) 0:49:41.43
iso.3.6.1.2.1.1.4.0 = STRING: "Tim Robinson"
iso.3.6.1.2.1.1.5.0 = STRING: "Netonix_Switch"
iso.3.6.1.2.1.1.6.0 = STRING: "RG22 4RW"
iso.3.6.1.2.1.2.1.0 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.1.1 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.1.2 = INTEGER: 2
iso.3.6.1.2.1.2.2.1.1.3 = INTEGER: 3
iso.3.6.1.2.1.2.2.1.1.4 = INTEGER: 4
iso.3.6.1.2.1.2.2.1.1.5 = INTEGER: 5
iso.3.6.1.2.1.2.2.1.1.6 = INTEGER: 6
iso.3.6.1.2.1.2.2.1.2.1 = STRING: "Port 1 - Trunk to Cisco"
iso.3.6.1.2.1.2.2.1.2.2 = STRING: "Port 2 - Port 2"
iso.3.6.1.2.1.2.2.1.2.3 = STRING: "Port 3 - Port 3"
iso.3.6.1.2.1.2.2.1.2.4 = STRING: "Port 4 - Port 4"
iso.3.6.1.2.1.2.2.1.2.5 = STRING: "Port 5 - Airfiber 5X to Oakridge"
iso.3.6.1.2.1.2.2.1.2.6 = STRING: "Port 6 - Port 6"
iso.3.6.1.2.1.2.2.1.8.1 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.8.2 = INTEGER: 2
iso.3.6.1.2.1.2.2.1.8.3 = INTEGER: 2
iso.3.6.1.2.1.2.2.1.8.4 = INTEGER: 2
iso.3.6.1.2.1.2.2.1.8.5 = INTEGER: 1
iso.3.6.1.2.1.2.2.1.8.6 = INTEGER: 2
iso.3.6.1.2.1.2.2.1.10.1 = Counter32: 6264280
iso.3.6.1.2.1.2.2.1.10.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.10.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.10.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.10.5 = Counter32: 4291367263
iso.3.6.1.2.1.2.2.1.10.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.11.1 = Counter32: 4294955677
iso.3.6.1.2.1.2.2.1.11.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.11.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.11.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.11.5 = Counter32: 17070
iso.3.6.1.2.1.2.2.1.11.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.1 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.5 = Counter32: 0
iso.3.6.1.2.1.2.2.1.13.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.1 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.5 = Counter32: 0
iso.3.6.1.2.1.2.2.1.14.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.16.1 = Counter32: 43822102
iso.3.6.1.2.1.2.2.1.16.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.16.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.16.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.16.5 = Counter32: 2166431
iso.3.6.1.2.1.2.2.1.16.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.17.1 = Counter32: 4294944067
iso.3.6.1.2.1.2.2.1.17.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.17.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.17.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.17.5 = Counter32: 17376
iso.3.6.1.2.1.2.2.1.17.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.1 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.5 = Counter32: 0
iso.3.6.1.2.1.2.2.1.19.6 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.1 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.2 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.3 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.4 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.5 = Counter32: 0
iso.3.6.1.2.1.2.2.1.20.6 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.1.1 = STRING: "Port 1                                                        "
iso.3.6.1.2.1.31.1.1.1.1.2 = STRING: "Port 2                                                        "
iso.3.6.1.2.1.31.1.1.1.1.3 = STRING: "Port 3                                                        "
iso.3.6.1.2.1.31.1.1.1.1.4 = STRING: "Port 4                                                        "
iso.3.6.1.2.1.31.1.1.1.1.5 = STRING: "Port 5                                                        "
iso.3.6.1.2.1.31.1.1.1.1.6 = STRING: "Port 6                                                        "
iso.3.6.1.2.1.31.1.1.1.2.1 = Counter32: 13892
iso.3.6.1.2.1.31.1.1.1.2.2 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.2.3 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.2.4 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.2.5 = Counter32: 14
iso.3.6.1.2.1.31.1.1.1.2.6 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.3.1 = Counter32: 2290
iso.3.6.1.2.1.31.1.1.1.3.2 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.3.3 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.3.4 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.3.5 = Counter32: 7
iso.3.6.1.2.1.31.1.1.1.3.6 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.4.1 = Counter32: 669
iso.3.6.1.2.1.31.1.1.1.4.2 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.4.3 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.4.4 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.4.5 = Counter32: 1897
iso.3.6.1.2.1.31.1.1.1.4.6 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.5.1 = Counter32: 19
iso.3.6.1.2.1.31.1.1.1.5.2 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.5.3 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.5.4 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.5.5 = Counter32: 46
iso.3.6.1.2.1.31.1.1.1.5.6 = Counter32: 0
iso.3.6.1.2.1.31.1.1.1.6.1 = Counter64: 6264280
iso.3.6.1.2.1.31.1.1.1.6.2 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.6.3 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.6.4 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.6.5 = Counter64: 13177183
iso.3.6.1.2.1.31.1.1.1.6.6 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.7.1 = Counter64: 37735
iso.3.6.1.2.1.31.1.1.1.7.2 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.7.3 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.7.4 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.7.5 = Counter64: 17049
iso.3.6.1.2.1.31.1.1.1.7.6 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.8.1 = Counter64: 2290
iso.3.6.1.2.1.31.1.1.1.8.2 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.8.3 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.8.4 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.8.5 = Counter64: 7
iso.3.6.1.2.1.31.1.1.1.8.6 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.9.1 = Counter64: 13892
iso.3.6.1.2.1.31.1.1.1.9.2 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.9.3 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.9.4 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.9.5 = Counter64: 14
iso.3.6.1.2.1.31.1.1.1.9.6 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.10.1 = Counter64: 43822102
iso.3.6.1.2.1.31.1.1.1.10.2 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.10.3 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.10.4 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.10.5 = Counter64: 2166431
iso.3.6.1.2.1.31.1.1.1.10.6 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.11.1 = Counter64: 41619
iso.3.6.1.2.1.31.1.1.1.11.2 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.11.3 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.11.4 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.11.5 = Counter64: 15433
iso.3.6.1.2.1.31.1.1.1.11.6 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.12.1 = Counter64: 19
iso.3.6.1.2.1.31.1.1.1.12.2 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.12.3 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.12.4 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.12.5 = Counter64: 46
iso.3.6.1.2.1.31.1.1.1.12.6 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.13.1 = Counter64: 669
iso.3.6.1.2.1.31.1.1.1.13.2 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.13.3 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.13.4 = Counter64: 0
iso.3.6.1.2.1.31.1.1.1.13.5 = Counter64: 1897
iso.3.6.1.2.1.31.1.1.1.13.6 = Counter64: 0
iso.3.6.1.2.1.105.1.3.1.1.3.1 = INTEGER: 2
iso.3.6.1.2.1.105.1.3.1.1.3.2 = INTEGER: 2
iso.3.6.1.2.1.105.1.3.1.1.3.3 = INTEGER: 2
iso.3.6.1.2.1.105.1.3.1.1.3.4 = INTEGER: 2
iso.3.6.1.2.1.105.1.3.1.1.3.5 = INTEGER: 1
iso.3.6.1.2.1.105.1.3.1.1.3.6 = INTEGER: 2
iso.3.6.1.2.1.105.1.3.1.1.4.1 = Gauge32: 0
iso.3.6.1.2.1.105.1.3.1.1.4.2 = Gauge32: 0
iso.3.6.1.2.1.105.1.3.1.1.4.3 = Gauge32: 0
iso.3.6.1.2.1.105.1.3.1.1.4.4 = Gauge32: 0
iso.3.6.1.2.1.105.1.3.1.1.4.5 = Gauge32: 6
iso.3.6.1.2.1.105.1.3.1.1.4.6 = Gauge32: 0

### PR DESCRIPTION
**Pull request by laf**
Fixes #1466 

@gr0mit1 When this merges, if you can give us the output of:

```bash
snmpbulkwalk -v2c -c BHNetworks -M NETONIX-SWITCH-MIB -m mibs/netonix 172.16.201.1 fanTable
snmpbulkwalk -v2c -c BHNetworks -M NETONIX-SWITCH-MIB -m mibs/netonix 172.16.201.1 tempTable
snmpbulkwalk -v2c -c BHNetworks -M NETONIX-SWITCH-MIB -m mibs/netonix 172.16.201.1 voltageTable
```

### CODE REVIEW

File: includes/polling/os/netonix.inc.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+
```
Comment (f0o) [inline includes/polling/os/netonix.inc.php]: ?

Comment (laf) [inline includes/polling/os/netonix.inc.php]: I can remove that, it's just got nothing available at the moment to get things like serial, hardware out.

**Comment by f0o** [comment]
Can you remove the empty file and reference where the MIB is from please?

**Comment by laf** [comment]
File removed.

Mib is from: http://forum.netonix.com/firmware/wisp-switch-mib
Logo is taken from the website and resized: http://www.netonix.com/skin/frontend/base/default/images/media/logo.png

**Comment by f0o** [comment]
I think a rogue file got in this PR: `mibs/netonix/.NETONIX-SWITCH-MIB.swp`

**Comment by laf** [comment]
Well spotted - updated

**Pull request merged**